release/models/policy-forwarding/openconfig-pf-srte.yang

```diff
@@ -124,6 +154,14 @@ module openconfig-pf-srte {
         protocol, or interface to the device, or from other static
         entries configured on the system.";
     }
+
+    leaf explicit-null-label {
+      type explicit-null-label-type;
+      description
+        "Type of explicit null label that is being pushed on an unlabeled
+        IP packet. If unspecified, the decision of whether to push 
+        an Explicit NULL label on a given packet is a matter of local policy."
```

Comment:
Can we add a `reference` statement to the draft and the relevant section here?

Reply:
Done.